Health-Application-Saturday/.editorconfig

```diff
@@ -0,0 +1,244 @@
+root = true
+
+[*]
```

Comment:
Did you mean to add an editor config file? I want to make sure you understand what it is what it is doing before being merged in.

Reply:
I didn't mean to add an editor config file, this was done by accident while I was trying to add git ignore, I looked up it is for consistent code formatting, if not needed, we could set root=false right? Thank you for catching that.

Reply:
I just want to make sure you were aware. If you want it in there, that is fine.